tests/virt_autotest/guest_installation_run.pm

```diff
@@ -69,9 +69,6 @@ sub post_execute_script_assertion {
 sub run {
     my $self = shift;
 
-    select_console 'sol', await_console => 0;
-    use_ssh_serial_console;
-
```

Comment:
I added these two lines for supporting run this test module only without installing a host, but I did not test on s390. Sorry for the troubles. Could you put these two lines in is_x86() instead of removing them?

Reply:
Yep, moved these two lines in is_x86()  now.

Reply:
Thanks!